api/src/services/user.service.ts

```diff
@@ -718,25 +718,6 @@ export class UserService {
           confirmationUrl,
         );
       }
-
-      // Partner user that is given access to an additional jurisdiction
-    } else if (
-      forPartners &&
-      existingUser &&
-      'userRoles' in dto &&
-      existingUser?.userRoles?.isPartner &&
-      dto?.userRoles?.isPartner &&
-      this.jurisdictionMismatch(dto.jurisdictions, existingUser.jurisdictions)
-    ) {
-      const newJurisdictions = this.getMismatchedJurisdictions(
```

Comment:
suggestion(non-blocking): The function `getMismatchedJurisdictions` can also be cleaned up. There is one other instance of it being used but in that case we just need to know if there is a mismatch. So we could delete this function and re-implement the appropriate logic in jurisdictionMismatch